opencti-platform/opencti-front/src/private/components/common/stix_domain_objects/StixDomainObjectEditionOverview.jsx

```diff
@@ -359,7 +359,7 @@ const StixDomainObjectEditionContainer = (props) => {
                   }
                 />
               )}
-              {'aliases' in stixDomainObject && (
```

Comment:
How can we have both ?
Do we have a problem with the declaration of attributes which means that we have the two fields: aliases & x_opencti_aliases?

Reply:
seen